src/services/YotiService.ts

```diff
@@ -30,6 +30,14 @@ export class YotiService {
 
 	readonly validationHelper: ValidationHelper;
 
+	static validateEnvVars(): void {
```

Comment:
This is an interesting pattern, how come we haven't used the existing env var patterns that we already have?

Reply:
Looks like a chance done as part of refactoring this service. Limitation of doing this check here is that we would be erroring out due a env missing at later stage and not at the processor initialization like we do for all other services. We may have to re-think about this or leave it as is.

Reply:
I added it here to have the check in one place but happy to move it to the relevant processors